### ISSUE TITLE
Pls make it show folders only, before we select it 

### ISSUE THREAD

**Issue by igs-kumayu** [catscandrive/comfyui-imagesubfolders#7]
Pls make it show folders only, before we select it .
in my case, there is dozen of pics  in each folder, the list is hundreds items , too long to scroll

**Comment by lzhfdwu007** [comment]
I also have this requirement
T_T

**Comment by catscandrive** [comment]
I'll look into it.  As this is just a glorified fork, it might be a lot of work to change the UI, and I don't have comfyui set up anymore as my gpu died.

**Comment by catscandrive** [comment]
I don't think this can be done with what I have here.  It requires js changes that I don't know how to do.  There is a different "subfolder" image loader in the ui manager though.  Unfortunately that one requires you to manually type in the subfolder?  I have forked and changed that one though so that you can choose the subfolder, then the image list will only be the images in that subfolder:

https://github.com/catscandrive/comfyui-loadimagewithsubfolder

You'll have to install it via "install via git url" though.
